### DISCUSSION TITLE
Crisp Intergration

### DISCUSSION THREAD

**Discussion by gbourne1** [GitbookIO/community#43] [Feature Requests]
### What's your feature/idea?

Please describe here - in as much detail as you can - your idea.
The new integrations are great, but the only chat integration is Intercom. Would like to see [Crisp](https://crisp.chat/en/) as an additional chat integration. This means having the chat icon appear in the bottom right corner of the screen that the user clicks and opens a chat window.

### What's your desired outcome?

What would you like to be able to do, if your idea came to life?
Communicate with user directly from the Gitbook pages. Presently you need to have a link in the header that most users don't see and opens a new page instead of a chat window.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by MantisClone** [comment]
I asked a related question: https://github.com/orgs/GitbookIO/discussions/315

**Comment by addisonschultz** [comment]
Hey @gbourne1 - just wanted to give you a ping. We've recently launched our official Crisp integration! You can read more about it here: https://github.com/orgs/GitbookIO/discussions/652

Feel free to let us know if there's any questions 🥳
